modules/apps/object/object-service/src/main/java/com/liferay/object/internal/validation/rule/DDMObjectValidationRuleEngineImpl.java

```diff
@@ -52,6 +53,10 @@ public Map<String, Object> execute(
 				_ddmExpressionFactory.createExpression(
 					CreateExpressionRequest.Builder.newBuilder(
 						script
+					).withDDMExpressionFieldAccessor(
+						new ObjectEntryDDMExpressionFieldAccessor(
+							(Map<String, Object>)inputObjects.get(
+								"objectEntry"))
```

Comment:
@p-albuquerque @devpedrotavares please remember to rename here when this [commit](https://github.com/liferay-objects/liferay-portal/pull/1688/commits/23c0698c7d4699283dc7aff001e683b1b217ff41) is merged

Reply:
Understood, thank you.